packages/SwingSet/src/controller.js

```diff
@@ -254,7 +263,7 @@ export async function buildVatController(config, withSES = true, argv = []) {
   // Evaluate source to produce a setup function. This binds withSES from the
   // enclosing context and evaluates it either in a SES context, or without SES
   // by directly calling require().
-  async function evaluateToSetup(sourceIndex, filePrefix = undefined) {
+  async function evaluateToSetup(sourceIndex, tag = undefined) {
```

Comment:
Some of this looks like it's not anylogger related at all.  Am I misunderstanding?

Reply:
> Some of this looks like it's not anylogger related at all. Am I misunderstanding?

I'm configuring the `console` endowment to be redirected to anylogger.  Changing the tag will allow for filtering based on vat, etc (in the future).  The `filePrefix` was a little too low-level, instead it should be just a tag out of which the `filePrefix` is built.